app/SettledExpense.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class SettledExpense extends Model
```

Comment:
I thought of this too. Just add a settled column. But then, I was thinking that maybe Vince is planning to archive the settled expenses